### DISCUSSION TITLE
Welcome to Ubuntu-Tricks Discussions!

### DISCUSSION THREAD

**Discussion by iamstarlee** [iamstarlee/Ubuntu-Tricks#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eriszhang99** [comment]
Hi, everyone!

**Reply by iamstarlee** [reply]
Hi, eris!

**Comment by eriszhang99** [comment]
Can I do simulation in Ubuntu OS?

**Reply by iamstarlee** [reply]
Depend on your software, if they don't have Linux version, you can not do that.

**Comment by iamstarlee** [comment]
Test the discussion
